CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ TODO: v2 changes
 
 ## [Unreleased]
 
+## [2.4.1] - 2021-07-20
```

Comment:
2.5.0 surely, change of behaviour, errors in a new place, change of outputs.

Reply:
no change in behaviour imo, still doesn't push the gem in the same cases

Reply:
to be fair, I'm considering the output from the action to purely be a side-effect of running it.